src/ChipInput.js

```diff
@@ -554,7 +555,7 @@ class ChipInput extends React.Component {
                 isFocused: dataSourceConfig ? (this.state.focusedChip && this.state.focusedChip[dataSourceConfig.value] === value) : (this.state.focusedChip === value),
                 handleClick: () => this.setState({ focusedChip: value }),
                 handleRequestDelete: () => this.handleDeleteChip(value, i),
-                defaultStyle: styles.defaultChip
+                defaultStyle: Object.assign(styles.defaultChip, chipStyle)
```

Comment:
Just a note here: This would modify `styles.defaultChip` every time `render` gets called.

Reply:
good one
had to change the order `Object.assign(chipStyle, styles.defaultChip)`
or use empty object as first elem `Object.assign({}, chipStyle, styles.defaultChip)`